python/GafferUI/GraphBookmarksUI.py

```diff
@@ -93,22 +93,22 @@ def appendPlugContextMenuDefinitions( nodeGraph, plug, menuDefinition ) :
 				}
 			)
 
-##########################################################################
-# Internal implementation
-##########################################################################
-
-def __getBookmarked( node ) :
+def getBookmarked( node ) :
 
 	return Gaffer.Metadata.value( node, "graphBookmarks:bookmarked" ) or False
 
-def __setBookmarked( node, bookmarked ) :
+def setBookmarked( node, bookmarked ) :
```

Comment:
I wonder what you think to these methods being a part of MetadataAlgo instead? We already have `setReadOnly()`/`getReadOnly()` which seem fairly equivalent in terms of defining a convention that will then be used in different parts of the UI. Just asking really, would be interested to know what other folks think too...